forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -2253,7 +2253,8 @@ public final String keywordsToText(final Collection<KeywordInterface> keywords)
                         || keyword.startsWith("Graft") || keyword.startsWith("Fading") || keyword.startsWith("Vanishing")
                         || keyword.startsWith("Afterlife") || keyword.startsWith("Hideaway") || keyword.startsWith("Toxic")
                         || keyword.startsWith("Afflict") || keyword.startsWith ("Poisonous") || keyword.startsWith("Rampage")
-                        || keyword.startsWith("Renown") || keyword.startsWith("Annihilator") || keyword.startsWith("Devour")) {
+                        || keyword.startsWith("Renown") || keyword.startsWith("Annihilator") || keyword.startsWith("Devour")
+                        || keyword.startsWith("For Mirrodin!")){
```

Comment:
this part should be moved to the Equip Section below, and it should be `|| keyword.equals("For Mirrodin")`

because now it is rendered twice

Reply:
Also this is the wrong section for "For Mirrodin"

this section is for "Keyword N" ones, Mirrodin is a SimpleKeyword